mod/Domestication/common/diplomacy_economy/domestication_diplo_costs.txt

```diff
@@ -1,3 +1,6 @@
+once = {
```

Comment:
Not sure about this one. It might overwrite the 'once' block in 00_diplomacy_economy.txt

Reply:
/shrug.  We can see how it goes and fix if necessary.

Reply:
yup, I tested it, it does. removed.